dev-python/b2sdk/files/b2sdk-1.2.0_skip-license-header-test.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/test/static/test_licenses.py b/test/static/test_licenses.py
+index babda33..b682f6d 100644
+--- a/test/static/test_licenses.py
++++ b/test/static/test_licenses.py
+@@ -14,6 +14,7 @@ from itertools import islice
+ import pytest
+ 
+ 
++@pytest.mark.skip(reasot="This is an upstream issue, not gentoo")
```

Comment:
- reasot? :)
- Got a link? Or do you mean that it's like a linter => the result doesn't matter to us?

Reply:
hrm, I wasn't running this in a "clean" environment, now that I am I am seeing different test errors, even disabling this patch. I'll dig into this.

Reply:
Turns out this was a red herring, re-enabled the test and everything worked fine.